src/multibody/geometry.hxx

```diff
@@ -81,6 +81,28 @@ namespace pinocchio
   , outerObjects (other.outerObjects)
   {}
 
+  inline GeometryData& GeometryData::operator=(const GeometryData & other)
```

Comment:
Why do you need to explicit it?

Reply:
Since the copy constructor is defined we also need to define the operator= (`-Wdeprecated-copy`). There are a few more examples e.g. `LieGroupBase` that I haven't fixed yet.